README.md

```diff
@@ -1,7 +1,5 @@
 ![PyBaMM_logo](https://user-images.githubusercontent.com/20817509/107091287-8ad46a80-67cf-11eb-86f5-7ebef7c72a1e.png)
 
-#
```

Comment:
This should not be removed. It adds a line between the PyBaMM logo and the badges. Makes the README looks good when rendered haha

Reply:
these mistakes might have made while merging with develop as I remember that I haven't made any changes other than colab badges in readme.
Strange, will fix them. Sorry 😅